### PR TITLE
Added Will's bio

### PR DESCRIPTION
**Pull request by flowersw**
@chrisarusso @AnneTee 

Comments are welcome!

### CODE REVIEW

File: _team/will-flowers.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Will Flowers
+layout: team
+job_title: Web Developer Intern
+job_description: Will is a novice web developer at Savas, learning from a growing team and contributing to a variety of clients.
```
Comment (AnneTee) [inline _team/will-flowers.md]: @flowersw I'd change "a novice web developer" to "an intern," and "clients" to "projects"

File: _team/will-flowers.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+photo: "assets/img/team/cards/will-card.jpg"
+card: "assets/img/team/will-flowers.jpg"
```
Comment (AnneTee) [inline _team/will-flowers.md]: @flowersw It looks like these are reversed. The card image should be the wide hobby image. So, you'll need to switch the filenames of the images, then switch the photo and card image urls in lines 6 and 7 of will-flowers.md. I'm not sure if i explained that well, so let me know if it doesn't make sense!

Comment (flowersw) [inline _team/will-flowers.md]: I think that makes sense

**Comment by AnneTee** [comment]
Nice @flowersw! @chrisarusso once you've reviewed this can be merged in. :sparkles:

**Comment by chrisarusso** [comment]
Reviewed: :dancers: !

**Pull request merged**